CHANGELOG.md

```diff
@@ -1,5 +1,66 @@
 ### [Versions](https://mui.com/versions/)
 
+## 5.2.2
+
+<!-- generated comparing v5.2.1..master -->
+
+_Nov 29, 2021_
+
+A big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- ♿️ Improved accessibility of `Snackbar` and `TextField` in `@mui/material` (#29782) (#29850) (#29852) @eps1lon.
```

Comment:
I think it's worth highlighting the support for `sx` inside the `styled()` utility. The previous version was released kinda anonymously, so we didn't have the chance to announce it. We could also do it in the tweet as well.